### PR TITLE
Threatmodel

### PR DESCRIPTION
**Pull request by fournet**
as discussed on [SCITT] SCITT threat model.
I still need to discuss with Yogesh how to integrate some of his comments.

### CODE REVIEW

**Review by ad-l — APPROVED** [review]

**Comment by yogeshbdeshpande** [comment]
@fournet : Thank you for this. I think, we should start Threat Model as a separate document to begin with. The reason been, I expect Supply Chain Security been a complex subject the number of threats and mitigation strategies will be a long document.

I would prefer a pointer to the document from the master Architecture document, then!

Once the document is in final shape post Community review, at that time we can make a final call, should we merge it to Main Arch. Doc or leave it separate.

**Comment by henkbirkholz** [comment]
> @fournet : Thank you for this. I think, we should start Threat Model as a separate document to begin with. The reason been, I expect Supply Chain Security been a complex subject the number of threats and mitigation strategies will be a long document.
> 
> I would prefer a pointer to the document from the master Architecture document, then!
> 
> Once the document is in final shape post Community review, at that time we can make a final call, should we merge it to Main Arch. Doc or leave it separate.

Considering the scale of the addition and the fact that stand-alone terminology or threat models are a bit frowned upon, I think we should merge this PR for now. If there is a need for a separate threat model I-D we can arrive at that conclusion in the WG and with AD and chair guidance.

Keeping the number of I-Ds the same during chartering is a pro and not a con, I think.

File: draft-birkholz-scitt-architecture.md [hunk 1]
```diff
@@ -158,34 +158,36 @@ The terms defined in this section have special meaning in the context of Supply
...
+For example, a statement may represent a Software Bill Of Materials (SBOM) that lists the ingredients of a software Artifact, or some endorsement or attestation about an Artifact.
 
 Claim:
 
 : an identifiable and non-repudiable Statement about an Artifact made by an Issuer. In SCITT, Claims are encoded as COSE signed objects; the payload of the COSE structure contains the Statement.
 
 Issuer:
 
-: creator of Claims submitted to a Transparency Service for Registration. The Issuer may be the owner or author of the Artifact, or a completely independent third party.
+: an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
: an entity that makes Claims about Artifacts in the supply chain. The Issuer may be the owner or author of the Artifact, or an independent third party such as a reviewer or an endorser.
```

File: draft-birkholz-scitt-architecture.md [hunk 2]
```diff
@@ -158,34 +158,36 @@ The terms defined in this section have special meaning in the context of Supply
...
 
 Feed:
 
-: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Claims about the same Artifact.
- In COSE, Feed is one header attributes in the protected header of the Envelope.
+: an identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Claims about the same Artifact.
+In COSE, Feed is one header attributes in the protected header of the Envelope.
 
-Registry:
+Registry (or Log, or Ledger):
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
Registry:
```

File: draft-birkholz-scitt-architecture.md [hunk 3]
```diff
@@ -158,34 +158,36 @@ The terms defined in this section have special meaning in the context of Supply
...
 
-: the verifiable data structure that stores Claims in a transparency service. SCITT supports multiple Registry formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.
+: the verifiable append-only data structure that stores Claims in a transparency service. SCITT supports multiple Registry and Receipt formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
: the verifiable append-only data structure that stores Claims in a Transparency Service often referred to by the synonym log or ledger. SCITT supports multiple Registry and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees.
```

File: draft-birkholz-scitt-architecture.md [hunk 4]
```diff
@@ -158,34 +158,36 @@ The terms defined in this section have special meaning in the context of Supply
...
 
-Transparency Service:
+Transparency Service (or Notary):
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
Transparency Service:
```

File: draft-birkholz-scitt-architecture.md [hunk 5]
```diff
@@ -158,34 +158,36 @@ The terms defined in this section have special meaning in the context of Supply
...
 
 : the entity that maintains and extends the Registry, and endorses its state. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry . The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
: an entity that maintains and extends the Registry and endorses its state often referred to its synonym Notary. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry . The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
```

File: draft-birkholz-scitt-architecture.md [hunk 6]
```diff
@@ -195,15 +197,25 @@ Receipt:
...
+: the pre-condition enforced by the TS before registering a Claim,
+based on its Envelope (notably the identity of its Issuer)
+and on prior claims already in the Registry.
 
 Transparent Claim:
 
 : a Claim that is augmented with a Receipt of its registration. A Transparent Claim remains a valid Claim (as the Receipt is carried in the countersignature), and may be registered again in a different TS.
 
-Verifier:
+Verifier (or Consumer):
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
Verifier:
```

File: draft-birkholz-scitt-architecture.md [hunk 7]
```diff
@@ -195,15 +197,25 @@ Receipt:
...
+
+: an entity that consumes Transparent Claims, verifying their proofs and inspecting their Statements, either before using their Artifacts, or later to audit their provenance on the supply chain.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
: an entity that consumes Transparent Claims (a specialization of Claim Consumer), verifying their proofs and inspecting their Statements, either before using their Artifacts, or later to audit their provenance on the supply chain.
```

File: draft-birkholz-scitt-architecture.md [hunk 8]
```diff
@@ -195,15 +197,25 @@ Receipt:
...
 
-: the entity that consumes Transparent Claims, verifying their proofs and inspecting their Statements, either before using their Artifacts, or later to audit their supply chain.
+Auditor:
+
+: an entity that checks the correctness and consistency of all Claim registered by a TS.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
: an entity that checks the correctness and consistency of all Claim registered by a TS (a specialization of Claim Consumer).
```

File: draft-birkholz-scitt-architecture.md [hunk 9]
```diff
@@ -228,7 +240,7 @@ Reputable Issuers are thus incentivized to carefully review their Statements bef
                 |
                 v                      +------------------+
  Issuer    -> Statement    Envelope    | DID Key Manifest |
-                \           /          |  (decentraized)  |
+                \           /          |  (decentralized)  |
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-architecture.md]:
```suggestion
                \           /          | (decentralized)  |
```

**Review by henkbirkholz — CHANGES_REQUESTED** [review]

**Pull request merged**